interchaintest/swap/tokenswap_test.go

```diff
@@ -279,57 +281,66 @@ func TestTokenSwap(t *testing.T) {
 	_, _, _ = gaiaUser, neutronUser, osmoUser
```

Comment:
ehm, what?

Reply:
just to avoid go compiler complaining about unused variables